src/OSPSuite.Core/Domain/ParameterIdentifications/ParameterIdentificationRunMode.cs

```diff
@@ -30,92 +26,4 @@ public virtual void UpdatePropertiesFrom(IUpdatable source, ICloneManager cloneM
       {
       }
    }
-
-   public class MultipleParameterIdentificationRunMode : ParameterIdentificationRunMode
-   {
-      public int NumberOfRuns { get; set; } = Constants.DEFAULT_NUMBER_OF_RUNS_FOR_MULTIPLE_MODE;
-
-      public MultipleParameterIdentificationRunMode() : base(Captions.ParameterIdentification.RunModes.MultipleRuns, isSingleRun:false)
-      {
-      }
-
-      protected override ParameterIdentificationRunMode CreateClone()
-      {
-         return new MultipleParameterIdentificationRunMode();
-      }
-
-      public override void UpdatePropertiesFrom(IUpdatable source, ICloneManager cloneManager)
-      {
-         base.UpdatePropertiesFrom(source, cloneManager);
-         var sourceMultipleOptimizationOption = source as MultipleParameterIdentificationRunMode;
-         if (sourceMultipleOptimizationOption == null) return;
-         NumberOfRuns = sourceMultipleOptimizationOption.NumberOfRuns;
-      }
-   }
-
-   public class CategorialParameterIdentificationRunMode : ParameterIdentificationRunMode
-   {
-      public bool AllTheSame { get; set; }
-      public CalculationMethodCache AllTheSameSelection { get; } = new CalculationMethodCache();
-      public Cache<string, CalculationMethodCache> CalculationMethodsCache { get; } = new Cache<string, CalculationMethodCache>(onMissingKey: x => null);
-
-      public CategorialParameterIdentificationRunMode() : base(Captions.ParameterIdentification.RunModes.Category, isSingleRun: false)
-      {
-      }
-
-      public override void UpdatePropertiesFrom(IUpdatable source, ICloneManager cloneManager)
-      {
-         base.UpdatePropertiesFrom(source, cloneManager);
-         var sourceCategorialParameterIdentification = source as CategorialParameterIdentificationRunMode;
-         if (sourceCategorialParameterIdentification == null) return;
-         AllTheSame = sourceCategorialParameterIdentification.AllTheSame;
-         CalculationMethodsCache.Clear();
-         foreach (var keyValue in sourceCategorialParameterIdentification.CalculationMethodsCache.KeyValues)
-         {
-            CalculationMethodsCache[keyValue.Key] = keyValue.Value.Clone();
-         }
-      }
-
-      public IReadOnlyList<string> SelectedCategories
-      {
-         get { return CalculationMethodsCache.SelectMany(x => x.All()).Select(x => x.Category).Distinct().ToList(); }
```

Comment:
This was right here. So loading a categorial all the same will result in unselected UI